src/shared/components/pipeline-run-logs/logs/logs-utils.ts

```diff
@@ -154,8 +154,8 @@ export const getDownloadAllLogsCallback = (
         allLogs += await getTaskRunLog(
           workspace,
           namespace,
-          pipelineRunUID,
           taskRun?.metadata?.uid,
+          pipelineRunUID,
```

Comment:
since you've changed the order of parameters and arguments as well, doe this PR changes something?

Reply:
Still needs testing because not on all places it is changed. But I do not have access to test it so trying to get that.